.github/workflows/test-backend.yml

```diff
@@ -84,6 +81,12 @@ jobs:
           DJANGO_SETTINGS_ENVIRONMENT: CI
           DATABASE_URL: postgis://tvp:tvp@localhost:5432/tvp
 
+      - name: "Test for outdated graphql schema file"
+        working-directory: backend
+        run: poetry run python manage.py update_graphql_schema
+        env:
+          DJANGO_SETTINGS_ENVIRONMENT: AutomatedTests
+
```

Comment:
Is this meant to error only if the GraphQL schema cannot be generated? This should be handled simply by our test running, but maybe I'm missing something?